internal/task/scripts/format.go

```diff
@@ -28,12 +28,40 @@ percent=$2
 chunkfile_size=$3
 chunkfile_pool_dir=$4
 chunkfile_pool_meta_path=$5
+increment_format=$6
 
 mkdir -p $chunkfile_pool_dir
-$binary \
-  -allocatePercent=$percent \
-  -fileSize=$chunkfile_size \
-  -filePoolDir=$chunkfile_pool_dir \
-  -filePoolMetaPath=$chunkfile_pool_meta_path \
-  -fileSystemPath=$chunkfile_pool_dir
+
+if [ $increment_format == "true" ]
+then
+	rootdir=$(dirname $chunkfile_pool_dir) 
+	used=$(df $rootdir --output=used|tail -n 1)
+	avail=$(df $rootdir --output=avail|tail -n 1)
+	size=$(df $rootdir --output=size|tail -n 1)
+	used_percent=$(df $rootdir --output=pcent|tail -n 1|sed 's/%//'|xargs)
+    let minus=$percent-$used_percent
+    
+    if [ $minus -gt 0 ]
+    then
+      let file_size=$chunkfile_size+4096
+      let pre_allocate_num=$minus*$size*1024/100/$file_size
+
+      $binary \
+        -fileSize=$chunkfile_size \
+        -filePoolDir=$chunkfile_pool_dir \
+        -filePoolMetaPath=$chunkfile_pool_meta_path \
+        -fileSystemPath=$chunkfile_pool_dir \
+        -allocateByPercent=false \
+        -preAllocateNum=$pre_allocate_num
+    fi  
+else
+    $binary \
+      -allocatePercent=$percent \
+      -fileSize=$chunkfile_size \
+      -filePoolDir=$chunkfile_pool_dir \
+      -filePoolMetaPath=$chunkfile_pool_meta_path \
+      -fileSystemPath=$chunkfile_pool_dir
+fi
+
+
```

Comment:
The parameters of curve_format is incremental precent default and seems to be unnecessary to modify the script.